build/DependencyVersions.props

```diff
@@ -6,7 +6,8 @@
     <MsBuildPackagesVersion>0.1.0-preview-00028-160627</MsBuildPackagesVersion>
     <DependencyModelVersion>1.0.3</DependencyModelVersion>
     <PlatformAbstractionsVersion>1.0.3</PlatformAbstractionsVersion>
-    <NuGetVersion>4.0.0-rtm-2323</NuGetVersion>
+    <!-- non-official NuGet build taken from https://github.com/nuget/nuget.client/tree/release-4.1.0-netstandard2.0 to contain "2.0" TFMs -->
```

Comment:
This build is from a branch that isn't an official NuGet dev branch.  It is branched off of the official branch and then @ericstj made a change do do the netcoreapp2.0 => netstandard2.0 mapping.

The point of the comment is to tell people that we are using basically a "private" NuGet build and to not take a new NuGet build that doesn't have the netstandard2.0 mapping.

Maybe we could add a test that restoring a netcoreapp2.0 app that references netstandard2.0 works?